configs/tasks/rearrangpick_replica_cad.yaml

```diff
@@ -0,0 +1,162 @@
+ENVIRONMENT:
```

Comment:
`rearrangepick` not `rearrangpick` preferably.

Reply:
I'm not sure how that file got in the commit, removed the file entirely.